frontend/src/metabase/redux/ui.ts

```diff
@@ -0,0 +1,15 @@
+import { createAction, handleActions } from "metabase/lib/redux";
+
+export const SET_OPEN_MODAL = "metabase/ui/SET_OPEN_MODAL";
+export const CLOSE_MODAL = "metabase/app/CLOSE_MODAL";
```

Comment:
these should both probably be either `ui` or `app`, right?

Reply:
yeah